editoast/src/views/layers.rs

```diff
@@ -301,7 +301,9 @@ mod tests {
             ("http://localhost:8090/test", "http://localhost:8090/test/"),
             ("http://localhost:8090/test/", "http://localhost:8090/test/"),
         ] {
-            std::env::set_var("ROOT_URL", root_url);
+            unsafe {
```

Comment:
I'd say it would be nice to have a best practice of adding a comment when we have a `unsafe` in code (the same than with `#[allow(...)]`). And actually, it's not fine in this case because we usually run tests in parallel threads, and this is exactly the case for non-being safe. Sure, we could say that this is the only test that actually do that (but who will detect if another test is added?). Another possible solution would be to rely on something like `cargo-nextest` which parallelize tests with processes (not threads).

It seems that we already had the problem anyway so I'm fine with a comment justifying that it's the only test like this.

Reply:
Yes, as stated in my last sentence.

> It seems that we already had the problem anyway so I'm fine with a comment justifying that it's the only test like this.